src/Features/VisualBasic/Portable/NavigationBar/VisualBasicNavigationBarItemService.vb

```diff
@@ -6,27 +6,20 @@ Imports System.Collections.Immutable
 Imports System.Composition
 Imports System.Threading
 Imports Microsoft.CodeAnalysis
-Imports Microsoft.CodeAnalysis.Editor.Extensibility.NavigationBar
```

Comment:
same as C#, this is the 'compute items' portion of the feature broken out and moved down to the features layer.